README.md

```diff
@@ -76,6 +76,8 @@ Flags:
                                   with store.
       --insecure                  Send gRPC requests via plaintext instead of
                                   TLS.
+      --ticker-duration           Duration between batcher client writes.  
```

Comment:
Maybe `--batch-group-interval`? It's also always good to say in the help text what the default value is.

Reply:
I think considering the lingo we used already, I think `--batch-write-interval` would be a better fit.